src/components/CandidateAvatar/CandidateAvatar.tsx

```diff
@@ -24,11 +24,11 @@ const CandidateAvatar = (props: Props) => {
             return 'warning';
         }
         return 'good';
-    }
+    };
 
     const { imageUrl, matchPercentage, showPercentageOnHover, size, ...rest } = props;
 
-    const constrainedSize = Math.round(Math.max(32, size));
+    const constrainedSize = Math.round(Math.max(32, size!));
```

Comment:
I have typescript tip here in my IDE `Forbidden non-null assertion.`.
Can we use here Number(size)? That will be more clear.